src/AspNetCoreModuleV2/CommonLib/debugutil.h

```diff
@@ -2,19 +2,75 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 #pragma once
-#define ASPNETCORE_DEBUG_FLAG_INFO          0x00000001
-#define ASPNETCORE_DEBUG_FLAG_WARNING       0x00000002
-#define ASPNETCORE_DEBUG_FLAG_ERROR         0x00000004
+#include "stdafx.h"
+#include "dbgutil.h"
 
-extern DWORD g_dwAspNetCoreDebugFlags;
+#define ASPNETCORE_DEBUG_FLAG_INFO          DEBUG_FLAG_INFO
+#define ASPNETCORE_DEBUG_FLAG_WARNING       DEBUG_FLAG_WARN
+#define ASPNETCORE_DEBUG_FLAG_ERROR         DEBUG_FLAG_ERROR
+#define ASPNETCORE_DEBUG_FLAG_CONSOLE       0x00000008
```

Comment:
w3wp does not have console

Reply:
Sure, enabling this flag there would not do anything.

Reply:
Can you verify logs are debug logs are printed for both IIS and IISExpress to a file?

Reply:
Not really my intent at this point. We enable stdout capture for small periods of time, it won't capture all the logs. The intent here is to be able to see logs from functional tests first.

Reply:
I'm just concerned it will cause exceptions/AVs in w3wp.